pupil_src/launchables/player.py

```diff
@@ -424,10 +427,10 @@ def handle_notifications(n):
                 g_pool.new_seek = False
                 try:
                     new_frame = g_pool.capture.get_frame()
-                except EndofVideoFileError:
+                except (EndofVideoFileError, IndexError):
```

Comment:
why are we hitting index error? Can we not catch this in Video and fake capture and raise as EndofVideoFileError maybe rename to EndofVideoError while we are at it?